CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(threadpool)
+
+set(CMAKE_CXX_STANDARD 11)
+add_compile_options(-Wall)
+
+link_libraries(pthread)
```

Comment:
Neither does this. CMake has special macros for threading support.

Reply:
I'm sure that we should link pthread in cmakefile manually on UNIX platform.
And, on macox it's necessary.